Task01/Task01/Program.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace Task01
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Title = "Тучкины штучки";
+
+            Console.BackgroundColor = ConsoleColor.Gray;
+            Console.ForegroundColor = ConsoleColor.Magenta;
+            Console.WriteLine("'Тучкины штучки'" + " Владимир Маяковский");
+
+            Console.BackgroundColor = ConsoleColor.Black;
+            Console.WriteLine(" ");
+
+            Console.BackgroundColor = ConsoleColor.Gray;
+            Console.ForegroundColor = ConsoleColor.DarkGreen;
+            Console.WriteLine("Плыли по небу тучки.\r\nТучек - четыре штучки:\r\nот первой до третьей - люди;\r\nчетвертая была верблюдик.");
+
+            Console.BackgroundColor = ConsoleColor.Black;
+            Console.WriteLine(" ");
+
+
+            Console.BackgroundColor = ConsoleColor.Gray;
+            Console.ForegroundColor = ConsoleColor.Magenta;
+            Console.WriteLine("К ним, любопытством объятая,\r\nпо дороге пристала пятая,\r\nот нее в небосинем лоне\r\nразбежались за слоником слоник.");
+
+            Console.BackgroundColor = ConsoleColor.Black;
+            Console.WriteLine(" ");
+
+            Console.BackgroundColor = ConsoleColor.Gray;
+            Console.ForegroundColor = ConsoleColor.DarkGreen;
+            Console.WriteLine("И, не знаю, спугнула шестая ли,\r\nтучки взяли все - и растаяли.\r\nИ следом за ними, гонясь и сжирав,\r\nсолнце погналось - желтый жираф.");
```

Comment:
Если вы хотите задать текст из нескольких строк как строковую константу, излишне сначала возвращать точку вывода в начало строки(`\r`). Достаточно сразу перейти на новую строку (`\n`).
```suggestion
            Console.WriteLine("Плыли по небу тучки.\nТучек - четыре штучки:\nот первой до третьей - люди;\nчетвертая была верблюдик.");

            Console.BackgroundColor = ConsoleColor.Black;
            Console.WriteLine(" ");


            Console.BackgroundColor = ConsoleColor.Gray;
            Console.ForegroundColor = ConsoleColor.Magenta;
            Console.WriteLine("К ним, любопытством объятая,\nпо дороге пристала пятая,\nот нее в небосинем лоне\nразбежались за слоником слоник.");

            Console.BackgroundColor = ConsoleColor.Black;
            Console.WriteLine(" ");

            Console.BackgroundColor = ConsoleColor.Gray;
            Console.ForegroundColor = ConsoleColor.DarkGreen;
            Console.WriteLine("И, не знаю, спугнула шестая ли,\nтучки взяли все - и растаяли.\nИ следом за ними, гонясь и сжирав,\nсолнце погналось - желтый жираф.");
```